src/tree.js

```diff
@@ -120,7 +120,7 @@ export const resolvers = {
   },
 
   NodeChild: {
-    id: root => root.cell_id,
+    id: root => root.cell_id.split(",").map(item => item.trim()),
```

Comment:
Since you also have this line for Node as well, it might be worthwhile to refactor it out into a separate function and have these lines call that.

```suggestion
const formatIDStringToList = (idStr) => idStr.split(",").map(item => item.trim())

    id: root => formatIDStringToList(root.cell_id),
```